style.css

```diff
@@ -199,13 +203,21 @@ ul li {
   color: var(--work-details);
 }
 
+.card-content ul {
+  float: right;
+}
+
 .work-details {
   display: flex;
   align-items: center;
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

There are multiple ways of achieving this, one of them is creating a class `"flex"`:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓